### PR TITLE
Create and Edit erea for diggers

### PR DESCRIPTION
**Pull request by niklas2106**
I add the method new, create, edit and update to the diggers. 
So now you can create new diggers, where you can put in the name, technical specification, start-date, end-date and day rate. You also can upload the photo. Bit i think we have to work on where these data is stored. 

Here you can see it: 
image

### CODE REVIEW

File: app/models/digger.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class Digger < ApplicationRecord
   belongs_to :user
   has_many :bookings
+  validates :name, presence: true
+  validates :technical_specification, presence: true
+  validates :day_rate, numericality: true, presence: true
```
Comment (maxwrf) [inline app/models/digger.rb]: should we validate the date?

File: app/views/diggers/_from.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div class="container">
```
Comment (maxwrf) [inline app/views/diggers/_from.html.erb]: what file is it? _from? do you want to create a reusable component? than go in a new folder shared and call it form

File: app/models/digger.rb (hunk 1 above)
Comment (maxwrf) [reply app/models/digger.rb]: you can also add a individual validation function to check that the start date is smaller than the end date

File: app/views/diggers/_from.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  <%= simple_form_for(digger) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :technical_specification %>
+  <%= f.input :start_date %>
+  <%= f.input :end_date %>
+  <%= f.input :day_rate %>
+  <%= f.input :photo %>
```
Comment (maxwrf) [inline app/views/diggers/_from.html.erb]: why is this photo upload different to the one in the two other forms?

File: app/controllers/diggers_controller.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+class DiggersController < ApplicationController
+  def new
+  @digger = Digger.new
+  end
+
+  def create
+    @digger = Digger.new(digger_params)
+    @user = current_user.id
```
Comment (maxwrf) [inline app/controllers/diggers_controller.rb]: is this working??

File: app/controllers/diggers_controller.rb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    @digger.user_id = @user
```
Comment (maxwrf) [inline app/controllers/diggers_controller.rb]: this needs to be @digger.user = @user

File: app/views/diggers/_from.html.erb (hunk 3 above)
Comment (niklas2106) [reply app/views/diggers/_from.html.erb]: i do not use this form

Comment (maxwrf) [reply app/views/diggers/_from.html.erb]: then delete it

File: app/views/diggers/_from.html.erb (hunk 2 above)
Comment (niklas2106) [reply app/views/diggers/_from.html.erb]: i delete this one

File: app/models/digger.rb [hunk 6]
```diff
@@ -1,4 +1,16 @@
...
+  validate :end_after_start
```
Comment (maxwrf) [inline app/models/digger.rb]: what is end after start? Cant we only validate instance variables?

**Pull request merged**